#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_fix_cmdir.dpatch by Mario Limonciello <superm1@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add correct driver requirements for Command IR

@DPATCH@
diff -urNad lirc-0.8.2~/doc/lirc.hwdb lirc-0.8.2/doc/lirc.hwdb
--- lirc-0.8.2~/doc/lirc.hwdb	2007-06-09 14:38:18.000000000 -0500
+++ lirc-0.8.2/doc/lirc.hwdb	2007-07-12 17:53:01.000000000 -0500
@@ -113,7 +113,7 @@
 ATI/NVidia/X10 I & II RF Remote;atiusb;lirc_dev lirc_atiusb;hw_default;atiusb/lircd.conf.atiusb;
 ATI/NVidia/X10 RF Remote (userspace);atilibusb;none;hw_atilibusb;atiusb/lircd.conf.atilibusb;
 Creative USB IR Receiver (SB0540);sb0540;none;hw_sb0540;creative/lircd.conf.alsa_usb;
-COMMANDIR USB Transceiver;cmdir;lirc_dev lirc_cmdir;hw_default;;
+COMMANDIR USB Transceiver;cmdir;lirc_dev commandir lirc_cmdir;hw_default;;
 Dign HV5 HTPC IR/VFD Module;sasem;lirc_dev lirc_sasem;hw_default;sasem/lircd.conf.sasem;
 DViCO USB Remote;dvico;none;hw_dvico;dvico/lircd.conf.fusionHDTV;
 Home Electronics Tira USB device;tira;none;hw_tira;;
